summarize/src/main.rs

```diff
@@ -18,6 +18,10 @@ struct Opt {
     /// Writes the analysis to a json file next to <file_prefix> instead of stdout
     #[structopt(long = "json")]
     json: bool,
+
+    /// Filters the output rows where "% of total time" > "percent-above"
+    #[structopt(short = "pa", long = "percent-above", default_value = "1.0")]
```

Comment:
I think we shouldn't filter results currently because I'm not sure we have enough data to make a good decision at this point. So I think this should be set to `default_value = "100.0"`. 

After this PR lands, then I think we can experiment with compiling a number of crates (small, medium, and large) to see if there's a good number that makes sense as a default (perhaps that will even be `1.0`)